Source/santad/SNTEventLog.m

```diff
@@ -29,6 +29,10 @@
 @interface SNTEventLog ()
 @property NSMutableDictionary *detailStore;
 @property dispatch_queue_t detailStoreQueue;
+
+@property NSMutableDictionary *userNameMap;
```

Comment:
Add a comment to userNameMap and groupNameMap. Something like:
Do not access directly, this is used for caching. Only access from the nameMapQueue.